src/components/templates/templatePost/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import Layout from '../layout/'
+import Section from '../../templates/section'
+import { NoticiaProps } from '../../../types/types'
+import Post from '../../organisms/post'
+
+type TemplatePostProps = {
+  noticia: NoticiaProps | undefined
+}
+
+const TemplatePost = ({
+  noticia
+}: TemplatePostProps) => {
+  return (
+    <Layout>
+      {(noticia === undefined) &&
```

Comment:
noticia pode ser vazia?

Reply:
pode ser. se o cara mudar o URL ele retorna que a notícia não foi encontrada